tests/helpers/errors.py

```diff
@@ -3,16 +3,6 @@ def get_error_test_cases(errors):
 
 
 ERRORS = {
-    'invalid-parameters': {
```

Comment:
why did you remove this?

Reply:
It's not used

Reply:
this error wasn't actually being tested before, so instead of removing it I fixed it and included it in the tests to make sure we raise a 400 response properly

Reply:
correct me if I'm wrong, but in the test you're trying to get a test data by the provided key in `@pytest.mark.parametrize`. and once you deleted test data, this function might return `None` and it might be the test is not failing eventually

Reply:
'invalid-parameters' was not included in the parameters before

Reply:
But now I added it to the parameters and restored it in the errors file so we can test it too.